Jenkinsfile

```diff
@@ -67,9 +67,10 @@ pipeline {
                 stash name: 'repo-amd64',includes: 'rook-amd64.tar,build/common.sh,_output/tests/linux_amd64/,_output/charts/,tests/scripts/'
                 script{
                     def data = [
-                        "aws_1.6": "v1.6.7",
+                        "gce_1.6": "v1.6.7",
                         "gce_1.7": "v1.7.8",
-                        "gce_1.8": "v1.8.0"
+                        "gce_1.8": "v1.8.2",
+                        "aws_rawdevice_1.7": "v1.7.8"
```

Comment:
i'd suggest we only have one test run per version of k8s. We should be able to run combinations of tests within that. In this case, that means aws could run on 1.7 with devices, and gce would run 1.6 and 1.8 with directories.

Reply:
let me change that, i have updated the installer such that the cluster will use all devices if there is a additional device one cluster, if nbot it will use directories.